src/enclave/httpbuilder.h

```diff
@@ -17,7 +17,7 @@ namespace enclave
     static HeaderMap default_headers()
     {
       HeaderMap headers;
-      headers["Content-Type"] = "application/json";
+      headers["content-type"] = "application/json";
```

Comment:
Why is this lowercase?

Reply:
For consistency. The `HeaderMap` we build in `httpparser.h` has lower-case keys (so the lookup is correctly case-insensitive), so we should make all our header keys lower-case. Otherwise we have to duplicate the lower-casing elsewhere, in our signature construction code.